BCStudentSoftwareDevTeam/lsf
# Add additional form search filters

## Issue
Fixes issue #442 

Right now sorting is not entirely intuitive, and if it is, it is not particularly dynamic. There may be 3 different fields stored in one column (what if I want to sort first name vs last name) but right now all we can do is sort by the text in column. We need to add our own ordering functionality so we can sort by whatever we deem necessary.

## Changes
- Disabled current built in sorting mechanisms because they will now break and are no longer needed
- Added 3 drop downs to select which column you want to sort by, what field in that column you want to sort by, and whether you want it ascending or descending.
- Pass back a string containing the field we want to order by to the backend
- Added in backend logic to sort by particular field instead of one field per column.
- Upgraded the current datatables package to the most up to date one.

## To Test
- Checkout `form_search_filters_442`
- Reset from backup data
- Go to the supervisor portal, experiment around with some queries and try and verify that every field and column work as expected.
